modules/bbp-simulation/src/main/resources/schemas/hbphackaton/simulation/analysis/v0.1.0.json

```diff
@@ -0,0 +1,56 @@
+{
+  "@context": [
+    "{{base}}/contexts/nexus/core/schema/v0.1.0",
+    {
```

Comment:
nexus/core/schema context could be replaced with neurosciencegraph/core/schema context and "nsg": "https://bbp-nexus.epfl.ch/vocabs/bbp/neurosciencegraph/core/v0.1.0/", removed from this schema